cvat/apps/annotation/annotation.py

```diff
@@ -102,12 +102,12 @@ def reset(self):
 
 class Annotation:
     Attribute = namedtuple('Attribute', 'name, value')
-    LabeledShape = namedtuple('LabeledShape', 'type, frame, label, points, occluded, attributes, group, z_order')
+    LabeledShape = namedtuple('LabeledShape', 'type, frame, label, points, occluded, attributes, group, z_order, annotation_type')
     LabeledShape.__new__.__defaults__ = (0, 0)
-    TrackedShape = namedtuple('TrackedShape', 'type, points, occluded, frame, attributes, outside, keyframe, z_order')
+    TrackedShape = namedtuple('TrackedShape', 'type, points, occluded, frame, attributes, outside, keyframe, z_order, annotation_type')
```

Comment:
According the migration file `annotation_type` field was added for `LabeledShape`, `LabeledImage` and `LabeledTrack` and I'm not sure that this field should be added for `TrackedShape`. Also it's a reason for the tests failrue.

Reply:
Thanks, I solved it already. While running the unit test I ran into another issue that seems to be related to LabelMe format (correct me if I'm wrong!). Here,  the logs attached:
![1](https://user-images.githubusercontent.com/59498234/77929386-3ca92d80-72c7-11ea-9c4a-7c5c28531047.png)
![2](https://user-images.githubusercontent.com/59498234/77929390-3dda5a80-72c7-11ea-9c20-7220fd486b3f.png)
![3](https://user-images.githubusercontent.com/59498234/77929400-3fa41e00-72c7-11ea-81d5-02f45db6cc3c.png)